src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -310,13 +312,71 @@ describe('DetailsFieldsComponent', () => {
       },
       shouldRestartEntry: false
     };
-
     expect(component.saveEntry.emit).toHaveBeenCalledWith(data);
   });
 
+  it('should not modify the start_date when start_hour has not been modified', () => {
+    const dateTest = moment().format('YYYY-MM-DD');
```

Comment:
change: 
 `dateTest` to `currentDate`
 `startHourTest` to `startHour`

I think is necessary use other name of variable per example  `dateTest` to `dateCurrent`

Reply:
Ok it's changed